src/main/java/net/gnomeffinway/depenizen/objects/MythicMobs/MythicMobsMob.java

```diff
@@ -0,0 +1,287 @@
+        }
+
+        // <--[tag]
+        // @attribute <mythicmob@mythicmob.entity>
+        // @returns dEntity
+        // @description
+        // Returns the dEntity for the MythicMob.
+        // @plugin Depenizen, MythicMobs
+        // -->
+        else if (attribute.startsWith("entity")) {
+            return new dEntity(getLivingEntity()).getAttribute(attribute.fulfill(1));
+        }
+
+        // <--[tag]
+        // @attribute <mythicmob@mythicmob.global_cooldown>
+        // @returns Element(Number)
+        // @description
+        // Returns the MythicMob's global cooldown.
+        // @plugin Depenizen, MythicMobs
+        // -->
+        else if (attribute.startsWith("global_cooldown")) {
+            return new Duration(mob.getGlobalCooldown()).getAttribute(attribute.fulfill(1));
+        }
+
+        // <--[tag]
+        // @attribute <mythicmob@mythicmob.type>
+        // @returns Element
+        // @description
+        // Always returns 'Mythic Mob' for MythicMob objects. All objects fetchable by the Object Fetcher will return the
+        // type of object that is fulfilling this attribute.
+        // @plugin Depenizen, MythicMobs
+        // -->
+        else if (attribute.startsWith("type")) {
+            return new Element("Mythic Mob").getAttribute(attribute.fulfill(1));
+        }
+
+        return new Element(identify()).getAttribute(attribute);
+    }
+
+    @Override
+    public void adjust(Mechanism mechanism) {
+        Element value = mechanism.getValue();
+
+        // <--[mechanism]
+        // @object MythicMob
+        // @name global_cooldown
+        // @input Element(Number)
+        // @description
+        // Sets global cooldown of the MythicMob.
+        // @tags
+        // <mythicmob@mythicmob.global_cooldown>
+        // -->
+        if (mechanism.matches("global_cooldown") && mechanism.requireInteger()) {
+            mob.setGlobalCooldown(value.asInt());
+        }
+
+        // <--[mechanism]
+        // @object MythicMob
+        // @name reset_target
+        // @input None
+        // @description
+        // Reset the MythicMob's target.
+        // @tags
+        // <mythicmob@mythicmob.target>
+        // -->
+        else if (mechanism.matches("reset_target")) {
+            mob.resetTarget();
+        }
+
+        // <--[mechanism]
+        // @object MythicMob
+        // @name target
+        // @input dEntity
+        // @description
+        // Sets MythicMob's target.
+        // @tags
+        // <mythicmob@mythicmob.target>
+        // -->
+        else if (mechanism.matches("target") && mechanism.requireObject(dEntity.class)) {
+            dEntity mTarget = dEntity.valueOf(value.asString());
+            if (mTarget == null || !mTarget.isValid() || mTarget.getLivingEntity() == null) {
+                return;
+            }
+            BukkitEntity target = new BukkitEntity(mTarget.getBukkitEntity());
+            mob.setTarget(target);
+        }
+
```

Comment:
reportInvalid() 
Also for futureproofing probably check for resisted registered properties